graylog2-server/src/main/java/org/graylog2/shared/journal/LocalKafkaJournal.java

```diff
@@ -790,14 +791,22 @@ protected void flushDirtyLogs() {
         }
     }
 
+    @Override
     public long getCommittedOffset() {
         return committedOffset.get();
     }
 
+    @Override
     public long getNextReadOffset() {
         return nextReadOffset;
     }
 
+    @Override
+    public void setNextReadOffset(long nextReadOffset) {
```

Comment:
Can't think of a case where we would want to set anything other than committed + 1. 
Should we go for resetToLastCommittedOffset(), which would do that by default, to avoid misuse?